spring-boot-project/spring-boot-tools/spring-boot-maven-plugin/src/main/java/org/springframework/boot/maven/BuildImageMojo.java

```diff
@@ -164,7 +164,11 @@ private BuildRequest getBuildRequest(Libraries libraries) {
 	}
 
 	private TarArchive getApplicationContent(Owner owner, Libraries libraries) {
-		ImagePackager packager = getConfiguredPackager(() -> new ImagePackager(getJarFile()));
+		File jarFile = getJarFile();
+		ImagePackager packager = getConfiguredPackager(() -> new ImagePackager(jarFile));
```

Comment:
The `packager` can be moved below the `if` as it doesn't seem to be used when the jar exists already, I guess.

Reply:
I assume someone is going to hack this change around a bit more, so it's probably re-arranging the proverbial deck chairs. But done ayway.